### DISCUSSION TITLE
Retroarch can't launch a game when achievements are turned on for a game that is not in the database

### DISCUSSION THREAD

**Discussion by dreams897** [OnionUI/Onion#1042] [Support]
### Checklist

- [X] I am reporting a bug in the Onion OS - Any issues with the closed source frontend ([MainUI](https://github.com/OnionUI/MainUI-issues)), or [RetroArch](https://github.com/libretro/RetroArch) or any of the cores should be directed to their respective repositories
- [X] I have reviewed the [Onion Wiki](https://github.com/OnionUI/Onion/wiki) and did not find the answer
- [X] I have searched the existing (open and closed) issues to ensure I do not log a duplicate

### Onions OS Version

Onion V4.2.0-RC

### Miyoo Firmware Version

202303262339

### Provide a clear and concise description of the issue

Not sure if this can be fixed, but Retroarch can't launch a game that is not in the achievements database if achievements are enabled and connected to WiFi. So far I have only tested this on GBC (Gambette) games. 

### Steps to Reproduce

Provide login information to Achievements in Retroarch and enable them --> Launch a game that is not in the Achievements database --> Will end up stuck on game BIOs boot screen

### Provide any additional context or information that may help us investigate your issue

Turning off WiFi fixes this since it cannot check the database

**Comment by Aemiii91** [comment]
> - [x] I am reporting a bug in the Onion OS - Any issues with the closed source frontend ([MainUI](https://github.com/OnionUI/MainUI-issues)), or [RetroArch](https://github.com/libretro/RetroArch) or any of the cores should be directed to their respective repositories

Did you read this before checking it off?

Did you try saving a game override with cheevos turned off?

**Comment by VaceCloud** [comment]
same problem here
